### ISSUE TITLE
How to resolve undefined reference to `SkyLogin_* ?

### ISSUE THREAD

**Issue by vinnitu** [msndevs/skylogin#1]
```
~/home/skylogin$ make
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/common.o src/common.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/crc.o src/crc.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/login.o src/login.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/objects.o src/objects.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/platform_unix.o src/platform_unix.c
src/platform_unix.c: In function ‘FillRndBuffer’:
src/platform_unix.c:267:7: warning: ignoring return value of ‘read’, declared with attribute warn_unused_result [-Wunused-result]
   read(fd, Buffer, 0x464);
       ^
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/random.o src/random.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/skylogin.o src/skylogin.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/uic.o src/uic.c
gcc -I include/ -fPIC -O3 -D NDEBUG   -c -o src/credentials.o src/credentials.c
gcc -I include/ -fPIC -O3 -D NDEBUG -shared `pkg-config --cflags --libs openssl` -o libskylogin.so src/common.o src/crc.o src/login.o src/objects.o src/platform_unix.o src/random.o src/skylogin.o src/uic.o src/credentials.o
```

```
~/home/skylogin/test$ make
gcc -I ../include/ -O3 -D NDEBUG   -c -o test.o test.c
gcc -I ../include/ -O3 -D NDEBUG -lskylogin -L.. -o test test.o
test.o: In function `main':
test.c:(.text.startup+0x2b): undefined reference to `SkyLogin_Init'
test.c:(.text.startup+0x43): undefined reference to `SkyLogin_LoadCredentials'
test.c:(.text.startup+0x56): undefined reference to `SkyLogin_CreateUICString'
test.c:(.text.startup+0x62): undefined reference to `SkyLogin_Exit'
test.c:(.text.startup+0x95): undefined reference to `SkyLogin_PerformLogin'
collect2: error: ld returned 1 exit status
make: *** [test] Error 1
```

**Comment by dequis** [comment]
Uh, works for me. What platform?

**Comment by vinnitu** [comment]
ubuntu x64 14.04

2015-08-26 18:08 GMT+03:00 dx notifications@github.com:

**Comment by vinnitu** [comment]
ubuntu x64 14.04

**Comment by dequis** [comment]
Fixed with a990fc52e4758fbf2a5aff3dc676859c0c4db212
